### ISSUE TITLE
The problem of quickly discarding out of sync can be referred to in the itemDropCycle MOD

### ISSUE THREAD

**Issue by smilepeak** [Shaosil/LethalCompanyMods-GeneralImprovements#115]
For the issue of quickly discarding out of sync, you can refer to the ItemDropCycle MOD. However, currently the ItemDropCycle MOD has an issue of dropping something and not seeing it, so I did not use it.

This MOD should be transferred to the next item list when discarded.

[The problem of quickly discarding out of sync can be referred to in the itemDropCycle MOD](https://thunderstore.io/c/lethal-company/p/boxofbiscuits97/ItemDropCycler/)

**Comment by NilaierMusic** [comment]
Yeah, this issue is quite common and also happens with [Quick Drop](https://github.com/ness77/Quick_Drop/issues/1) mod. You would probably need to implement some sophisticated networking logic for that to actually work for clients if the host doesn't have the same mod installed.
